src/web-server/json-formatter.js

```diff
@@ -0,0 +1,34 @@
+import fs from '../lib/fs';
+
+export default class {
```

Comment:
This is my fault, but probably neither of these formatter modules need to be a `class`, since there's no state being held.  Let's just return the functions themselves on the module.